pom.xml

```diff
@@ -872,6 +872,21 @@
 			<artifactId>xml-apis</artifactId>
 			<version>1.4.01</version>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-core</artifactId>
+			<version>2.9.6</version>
```

Comment:
could you put the common version in a property?
(by the way do you really need these 3 dependencies?)